lib/Image/Operation/Resize.php

```diff
@@ -49,7 +73,7 @@ public function filename( $src_filename, $src_extension ) {
 		if ( $this->h ) {
 			$h = $this->h;
 		}
-		$result = $src_filename.'-'.$w.'x'.$h.'-c-'.($this->crop ? $this->crop : 'f'); // Crop will be either user named or f (false)
+		$result = $src_filename . '-' . $w . 'x' . $h . '-c-' . ( $this->crop ? $this->crop : 'f' ); // Crop will be either user named or f (false)
```

Comment:
Let’s not add too many coding standard fixes. We wanted to do that in one swoop before releasing 2.x, because they might distract from the actual changes. 😉